qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/parser/template/sections/FormSection.java

```diff
@@ -0,0 +1,108 @@
+/*******************************************************************************
+* Copyright (c) 2023 Red Hat Inc. and others.
+* All rights reserved. This program and the accompanying materials
+* which accompanies this distribution, and is available at
+* http://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Contributors:
+*     Red Hat Inc. - initial API and implementation
+*******************************************************************************/
+package com.redhat.qute.parser.template.sections;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.redhat.qute.parser.html.HtmlParser;
+import com.redhat.qute.parser.template.ASTVisitor;
+import com.redhat.qute.parser.template.HtmlElement;
+import com.redhat.qute.parser.template.Parameter;
+import com.redhat.qute.parser.template.Section;
+import com.redhat.qute.parser.template.SectionKind;
+import com.redhat.qute.services.diagnostics.CollectInputSectionTextVisitor;
+
+/**
+ * Form section.
+ * 
+ */
+public class FormSection extends Section {
```

Comment:
FormSection  should extends CustomSection. Please move FomrSection in a renarde package.